### ISSUE TITLE
Why aren't new versions being released to package repos ?

### ISSUE THREAD

**Issue by prasadpilla** [jazzband/django-auditlog#316]
Hi Team, 

This is a great alternative to heavy audit log libraries like simple-history. We're trying to move to this after working with simple-history for an year and being frustrated with the schema overhead it brings. 

I see that there's active development on this, but the last released package was in 2019, and that version has ref to outdated django-dateutil version which makes it impossible to use. Wondering if there's gonna be a new version out soon?

**Comment by Linkid** [comment]
Hi ! With @hramezani, we will release a new version soon :). We are waiting fo #268 to be closed, first.

**Comment by msf-dev** [comment]
@prasadpilla, did you end up making the switch from simple-history to auditlog and if so, were you able migrate the simple-history data over?

**Comment by prasadpilla** [comment]
@msf-dev, yes we did end up moving, but we didn't bother migrating the existing data. The format is very different, we would've had to do it through a script ourselves.
